server/controller/user.controller.js

```diff
@@ -35,274 +34,233 @@ import { User } from '../database/models';
 
 const userController = express.Router();
 
-function createUser(email, password) {
+/**
+ * Create a User with email, password, is_admin default = FALSE
+ */
+userController.createUser = (email, password) => {
```

Comment:
Inconsistent placing (location) of where these helper functions are within a file. Located at the top in user.controller and located at the bottom in course.controller.